SConstruct

```diff
@@ -905,6 +912,19 @@ if os.pathsep == ';':
 env['extra_inc_dirs'] = [d for d in env['extra_inc_dirs'].split(os.pathsep) if d]
 env['extra_lib_dirs'] = [d for d in env['extra_lib_dirs'].split(os.pathsep) if d]
 
+# Add conda library/include paths (if applicable) to extra
+conda_prefix = os.environ.get("CONDA_PREFIX")
+if conda_prefix is not None:
+    if os.name == "nt":
+        conda_inc_dir = pjoin(conda_prefix, "Library", "include")
+        conda_lib_dir = pjoin(conda_prefix, "Library", env["libdirname"])
+    else:
+        conda_inc_dir = pjoin(conda_prefix, "include")
+        conda_lib_dir = pjoin(conda_prefix, env["libdirname"])
+    env["extra_inc_dirs"] = env["extra_inc_dirs"] + [conda_inc_dir]
+    env["extra_lib_dirs"] = env["extra_lib_dirs"] + [conda_lib_dir]
```

Comment:
```suggestion
    env["extra_inc_dirs"] = env["extra_inc_dirs"].append(conda_inc_dir)
    env["extra_lib_dirs"] = env["extra_lib_dirs"].append(conda_lib_dir)
```
Creating a list so that it can be added seems excessive here to me 😄